src/operators/convert.ts

```diff
@@ -104,7 +108,14 @@ export class ConvertOperator implements Operator {
 
     // if enumerate is set, try to coerce all strings into an equivalent numeric value
     if (enumerate) {
-      const enumerableProps = ConvertOperator.enumerableProperties(data[index]);
+      let enumerableProps = ConvertOperator.enumerableProperties(data[index]);
+      // if ignore properties are set, make sure those are filtered out
+      if (ignore) {
+        /* no matter what I did, breaking it up into multiple lines, or using curly braces, I couldn't get the line
```

Comment:
Ideally I'd prefer if we could remove the comments related to linting.  Perhaps we could just do `const filterPredicate = (key) => ignore.find((element:string) => element === key) === undefined);` and then `enumerableProps.filter(filterPredicate)`.  (Note code not testing, just typing it here for example.)

Reply:
I can remove the comments about the linting.  I even broke up the lines into separate inner functions, and it still complained, and would reformat the code on me.  I spent 30 minutes on it and gave up.  I'll remove the comment and see if your suggestion will fix it.

Reply:
That worked and I updated the pull request to remove the comment and the ignore line.